gymnasium/spaces/sequence.py

```diff
@@ -1,20 +1,25 @@
 """Implementation of a space that represents finite-length sequences."""
+from __future__ import annotations
+
 from collections.abc import Sequence as CollectionSequence
-from typing import Any, List, Optional, Tuple, Union
+from typing import Any
+from typing import Sequence as TypingSequence
+from typing import Tuple
 
 import numpy as np
+import numpy.typing as npt
 
-import gymnasium as gym
 from gymnasium.spaces.space import Space
 
 
-class Sequence(Space[Tuple]):
+class Sequence(Space[Tuple[Any, ...]]):
```

Comment:
We can use the generic here, and omit the `[Any, ...]` (so `Tuple[Any, ...]` -> `tuple`)

Reply:
I think that `Tuple[Any, ...]` provides more detail than `tuple` and `tuple[Any, ...]` will cause error pyright reports